File: data/src/main/java/com/github/gmetal/data/entity/GenreEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class GenreEntity {

    @JsonField
    public Integer id;
    @JsonField
    public String name;
}

File: data/src/main/java/com/github/gmetal/data/entity/GenreEntity.kt
package com.github.gmetal.data.entity

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class GenreEntity {

    @JsonField
    var id: Int? = null
    @JsonField
    var name: String? = null
}
